app/models/invenio_rdm_record_converter.rb

```diff
@@ -15,6 +15,7 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   def initialize(generic_file=nil)
     return unless generic_file
     @@header_lookup ||= HeaderLookup.new
+    @@funding_data ||= eval(File.read('app/models/concerns/galtersufia/generic_file/funding_data.txt'))
```

Comment:
Similar to what has been used for header lookup. Use a shared class variable and read from a hash stored in a text file with the magic of ruby's eval method.